src/components/MapView/Layers/BoundaryDropdown.tsx

```diff
@@ -299,8 +333,91 @@ function BoundaryDropdown({
       setSelectedBoundaries={newSelectedBoundaries => {
         dispatch(setSelectedBoundariesRedux(newSelectedBoundaries));
       }}
+      labelMessage={labelMessage}
     />
   );
 }
 
+export const SelectMenu = withStyles(() => {
+  return {
+    root: {
+      '& label': {
+        textTransform: 'uppercase',
+        letterSpacing: '3px',
+        fontSize: '11px',
+      },
+      '& svg': { color: 'white', fontSize: '1.25rem' },
+    },
+  };
+})(SimpleBoundaryDropdown);
```

Comment:
```suggestion
export const SelectMenu = withStyles(() => ({
    root: {
      '& label': {
        textTransform: 'uppercase',
        letterSpacing: '3px',
        fontSize: '11px',
      },
      '& svg': { color: 'white', fontSize: '1.25rem' },
    }
  }))(SimpleBoundaryDropdown);
```
May not be syntactically perfect, but hope you're getting the idea ;)

Reply:
Included in d5d3c84a2aecd2a652255af256f06d124848e901